site/src/components/DropdownArrows/DropdownArrows.tsx

```diff
@@ -15,12 +15,16 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }))
 
-export const OpenDropdown: FC = () => {
-  const styles = useStyles()
+interface ArrowProps {
+  margin?: boolean
+}
```

Comment:
We could share this typing with the `makeStyles` block above: `makeStyles<Theme, ArrowProps>`

Reply:
I also wonder if, instead of a `margin` prop, we add a `style` prop that we spread in the children. Makes things a bit more flexible, but just a suggestion!

Reply:
> We may want to consider keeping [link text blue](https://www.nngroup.com/articles/guidelines-for-visualizing-links/) for accessibility reasons.

@Kira-Pilot Yeah, so I changed this because I thought the color combination was bad and it was also going to pose a contrast issue, but I have been wondering if it's obvious enough. I think the chevron and placement helps. But I also think I'm having trouble making it blue or underlined because it's a not a navigation link. It's really a button in functionality. What do you think is the best way to style that? I guess Zenhub styles "Show 4 more" on this page like a link (bold and blue).

Reply:
Going to wait on a `style` prop here because I think this component will be revisited in future color work and it'll be easier to tell then what to do.

Reply:
@presleyp That makes sense! I agree with your logic and I don't think it's something we need to change right now. Let's keep it back of mind and look out for link inspo in the future.